src/components/ElvaYBPractice/styles/TopComponent.styled.ts

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components'
+
+export const TopComponent = styled.div`
+  color: hsl(179, 62%, 43%);
+  background-color: #ffffff;
+  width: 100%;
+  height: 35vh;
+  @media only screen and (max-width: 600px) {
+    display: flex;
+    flex-direction: column;
+    padding: 10%;
+    height: 30vh;
```

Comment:
Consider changing 30vh to auto so that content won't get cut off during mobile view